DEVELOPER_GUIDE.md

```diff
@@ -30,9 +30,9 @@ To get started, follow the [getting started section](https://github.com/opensear
 
 Next, run the following commands to copy the built code (snapshot) to a new folder in a different location. (This where you'll be running the OpenSearch service). Run this from the base directory of the OpenSearch fork you cloned above:
 ```bash
-export OPENSEARCH_HOME=~/<your-folder-location>/opensearch-*
-export OPENSEARCH_BUILD=distribution/archives/darwin-tar/build/install/opensearch-*
-cp -Rf $OPENSEARCH_BUILD $OPENSEARCH_HOME
+export OPENSEARCH_HOME=~/<your-folder-location>/opensearch-{VERSION-NUMBER}
```

Comment:
On this line there is another kind of substitution parameter `<your-folder-location>`, I think we should pick a single format.  However, I think it might be even more worthwhile to see about reframing this section to not require developers to think about the version number, how does that sound?

Maybe we can pull version numbers from a gradle command, or build up a script that will do the 'painful' stuff so that you don't have to when you want to get the instance up and running.

https://github.com/opensearch-project/security/blob/87520741566eaa4ea609441a1f53f3cef4a9674c/.github/workflows/ci.yml#L180-L182

Reply:
That's a great idea. Let me work on it. Thanks @peternied.